### PR TITLE
v1.76

### PR DESCRIPTION
**Pull request by LappySheep**
yea

### CODE REVIEW

File: lib/dungeonExpWeight.js [hunk 1]
```diff
@@ -14,18 +14,19 @@ const getDungeonExpWeight = cataXP => {
...
     let n;
+    let extra;
     let tempLevel = 0;
     if (cataXP < 569809640)
         n = 0.2 * (level / 50) ** 1.538679118869934;
     else {
         let part = 142452410;
         tempLevel = 50 + (cataXP - dungeonMaxXP) / part;
-        n = (1 + ((tempLevel - 50) / 50)) ** 2.967355422;
+        extra = (500 * ((exp - lv50) / part) ** (1/1.781925776625157));
```
Comment (muchnameless) [inline lib/dungeonExpWeight.js]:
```suggestion
        extra = (500 * ((cataXP - dungeonMaxXP) / part) ** (1/1.781925776625157));
```

File: lib/dungeonExpWeight.js [hunk 2]
```diff
@@ -14,18 +14,19 @@ const getDungeonExpWeight = cataXP => {
     }
 
     let n;
+    let extra;
     let tempLevel = 0;
     if (cataXP < 569809640)
         n = 0.2 * (level / 50) ** 1.538679118869934;
     else {
         let part = 142452410;
         tempLevel = 50 + (cataXP - dungeonMaxXP) / part;
```
Comment (muchnameless) [inline lib/dungeonExpWeight.js]:
```suggestion
```

File: lib/dungeonExpWeight.js [hunk 3]
```diff
@@ -14,18 +14,19 @@ const getDungeonExpWeight = cataXP => {
     }
 
     let n;
+    let extra;
     let tempLevel = 0;
     if (cataXP < 569809640)
```
Comment (muchnameless) [inline lib/dungeonExpWeight.js]:
```suggestion
    if (cataXP < dungeonMaxXP)
```

File: lib/dungeonExpWeight.js [hunk 4]
```diff
@@ -14,18 +14,19 @@ const getDungeonExpWeight = cataXP => {
...
         n = 0.2 * (level / 50) ** 1.538679118869934;
     else {
         let part = 142452410;
         tempLevel = 50 + (cataXP - dungeonMaxXP) / part;
-        n = (1 + ((tempLevel - 50) / 50)) ** 2.967355422;
+        extra = (500 * ((exp - lv50) / part) ** (1/1.781925776625157));
     }
 
     if (level !== 0) {
         if (cataXP < 569809640) return dungeonOverall * ((1.18340401286164044 ** (level + 1) - 1.05994990217254) * (1 + n));
```
Comment (muchnameless) [inline lib/dungeonExpWeight.js]:
```suggestion
        if (cataXP < dungeonMaxXP) return dungeonOverall * ((1.18340401286164044 ** (level + 1) - 1.05994990217254) * (1 + n));
```

File: lib/dungeonExpWeight.js (hunk 3 above)
Comment (muchnameless) [reply lib/dungeonExpWeight.js]: why not use the constant?

File: lib/dungeonExpWeight.js (hunk 1 above)
Comment (muchnameless) [reply lib/dungeonExpWeight.js]: `exp` and `lv50` are not defined in this scope

File: lib/dungeonExpWeight.js [hunk 5]
```diff
@@ -14,18 +14,19 @@ const getDungeonExpWeight = cataXP => {
     }
 
     let n;
+    let extra;
     let tempLevel = 0;
```
Comment (muchnameless) [inline lib/dungeonExpWeight.js]:
```suggestion
```

Comment (muchnameless) [reply lib/dungeonExpWeight.js]: this variable doesn't seem to be used anymore

**Review by muchnameless — CHANGES_REQUESTED** [review]

File: lib/dungeonExpWeight.js (hunk 1 above)
Comment (LappySheep) [reply lib/dungeonExpWeight.js]: oops thank you

File: lib/dungeonExpWeight.js (hunk 3 above)
Comment (LappySheep) [reply lib/dungeonExpWeight.js]: i dont remember why that was set to that oops

**Pull request merged**